commands/builtins/builtins_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/gomeeseeks/meeseeks-box/auth"
 	"github.com/gomeeseeks/meeseeks-box/commands"
 	"github.com/gomeeseeks/meeseeks-box/commands/builtins"
+	"github.com/gomeeseeks/meeseeks-box/commands/shell"
```

Comment:
not something to do now, but I think we should avoid including this import here and have some basic shell commands that we can use in any test, that would belong to the testingstubs package (which will be called mocks in the future).

Reply:
Agreed.